src/ort_genai_c.h

```diff
@@ -229,6 +248,13 @@ OGA_EXPORT void OGA_API_CALL OgaDestroyTokenizerStream(OgaTokenizerStream*);
  */
 OGA_EXPORT OgaResult* OGA_API_CALL OgaTokenizerStreamDecode(OgaTokenizerStream*, int32_t token, const char** out);
 
+OGA_EXPORT OgaResult* OGA_API_CALL OgaCreateTensorFromBuffer(void* data, const int64_t* shape_dims, size_t shape_dims_count, OgaElementType element_type, OgaTensor** out);
```

Comment:
Could you please add another one to create without data managed by Oga

Reply:
This is the API for that. Oga doesn't own the memory. I'll add documentation.

Reply:
sorry, i typed it wrongly. I meant to add one to create another with data managed by oga